.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: PowerShell Issues
+    url: https://github.com/PowerShell/PowerShell/issues/new
+    about: PowerShell issues or suggestions.
+  - name: Windows PowerShell Issues
+    url: https://support.microsoft.com/windows/send-feedback-to-microsoft-with-the-feedback-hub-app-f59187f8-8739-22d6-ba93-f66612949332
+    about: Windows PowerShell issues or suggestions.
+  - name: Support
+    url: https://github.com/PowerShell/PowerShell/blob/master/.github/SUPPORT.md
```

Comment:
should we have a SUPPORT.md file specifically for Crescendo?

Reply:
Good question. I think that depends on how much the support model differs here. It's not a bad idea to have, but if it doesn't actually differ then it will mean keeping it up-to-date in multiple places (which there's no mechanism for yet).